packages/plugins/src/api/index.js

```diff
@@ -111,19 +112,28 @@ const plugins = {};
  *
  * @return {WPPlugin} The final plugin settings object.
  */
-export function registerPlugin( name, settings ) {
+export function registerPlugin( name, settings, scope ) {
```

Comment:
Just wondering if `scope` should be added to the `settings` object instead with a default value of 'block-editor'?

Reply:
>Just wondering if scope should be added to the settings object

Great idea, cleaner than adding an argument.

>with a default value of block-editor?

With a default in place, that could present some confusion with `getPlugins`. When no arguments are supplied, would it return all plugins or just the default ones with scope `block-editor`? Also, `block-editor` is a sane default for now, but would it make sense in a future that includes Full Site Editing and any slotFills it may have?